_languages-frameworks/35-kotlin.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: details
+filename: kotlin
+name: Kotlin
+image: /tech-radar/assets/images/languages-frameworks/kotlin.png
+category: languages-frameworks
+ring: Adopt
+number: 35
+---
+
+# What is it ?
+Kotlin is a general purpose, free, open source programming language that can be used to develop a wide range of applications like server-side applications, web applications, among others. Kotlin offers features such as null safety, extension functions, and type inference to make code more concise and readable. Google announced official support for Kotlin on Android in 2017. Since then, it has gained popularity among Android developers as an alternative to Java.
```

Comment:
I think we normally spell "open-source" with an hyphen, so we should maintain that for consistency.

And I think more than an alternative, Kotlin is currently the goto language regarding Android Development at least, so I think we could emphasise that a bit more.